components/eam/bld/configure

```diff
@@ -335,6 +335,7 @@ GetOptions(
     "ccsm_seq"                  => \$opts{'ccsm_seq'},
     "cflags=s"                  => \$opts{'cflags'},
     "chem=s"                    => \$opts{'chem'},
+    "cldera_strat_volc=s"         => \$opts{'cldera_strat_volc'},
```

Comment:
Minor formatting request: the `=>` should be lined up with the other lines here.